### PR TITLE
Added @Disabled tests handling

### PR DESCRIPTION
**Pull request by sidelnikovmike**
### Context
This pull request fixes problem, that tests with annotation `@Disabled` not shown in report.
Allure shows skipped test only in case of assumption.

#### Checklist
- [x] [Sign Allure CLA][cla]
- [x] Provide unit tests

[cla]: https://cla-assistant.io/accept/allure-framework/allure2

### CODE REVIEW

**Comment by sskorol** [comment]
@sidelnikovmike seems like there're couple of PMD violations... I'd assume it failed on AllureJunit5.java - line 87 (missing final keyword on input args)... please run `gradlew check` locally to detect and fix them...

File: allure-junit-platform/src/main/java/io/qameta/allure/junit5/AllureJunit5.java [hunk 1]
```diff
@@ -72,15 +74,43 @@ public void executionStarted(final TestIdentifier testIdentifier) {
...
+                result.getLabels().add(new Label().withName(PACKAGE).withValue(source.getClassName()));
             });
 
             getLifecycle().scheduleTestCase(result);
             getLifecycle().startTestCase(uuid);
         }
     }
 
+    @Override
+    public void executionSkipped(TestIdentifier testIdentifier, String reason) {
```
Comment (sskorol) [inline allure-junit-platform/src/main/java/io/qameta/allure/junit5/AllureJunit5.java]: quality plugin will fail on missing `final` for inputs

File: allure-junit-platform/src/main/java/io/qameta/allure/junit5/AllureJunit5.java [hunk 2]
```diff
@@ -72,15 +74,43 @@ public void executionStarted(final TestIdentifier testIdentifier) {
...
+            final String uuid = tests.get();
+            final TestResult result = new TestResult()
+                    .withUuid(uuid)
+                    .withName(testIdentifier.getDisplayName())
+                    .withLabels(getTags(testIdentifier))
+                    .withHistoryId(getHistoryId(testIdentifier))
+                    .withStage(Stage.FINISHED)
+                    .withStatus(SKIPPED);
+
+            methodSource.ifPresent(source -> {
```
Comment (sskorol) [inline allure-junit-platform/src/main/java/io/qameta/allure/junit5/AllureJunit5.java]: just wondering, why do you need to create a local variable `methodSource`, if it's used only once?.. 89 line could be merged with this one I believe...

File: allure-junit-platform/src/main/java/io/qameta/allure/junit5/AllureJunit5.java (hunk 1 above)
Comment (sidelnikovmike) [reply allure-junit-platform/src/main/java/io/qameta/allure/junit5/AllureJunit5.java]: I'll look at checks, thanks

File: allure-junit-platform/src/main/java/io/qameta/allure/junit5/AllureJunit5.java (hunk 2 above)
Comment (sidelnikovmike) [reply allure-junit-platform/src/main/java/io/qameta/allure/junit5/AllureJunit5.java]: Don't be so wondering :) such things sometime happens.
Thanks, I'll fix this

**Comment by sidelnikovmike** [comment]
I'll now look at fail

File: allure-junit5/src/test/java/io/qameta/allure/junit5/AnnotationsTest.java [hunk 3]
```diff
@@ -53,67 +53,67 @@ void shouldProcessMethodLabels() {
...
+//        assertThat(testResults)
+//                .hasSize(1)
+//                .flatExtracting(TestResult::getLinks)
+//                .extracting(Link::getName)
+//                .contains(
+//                        "LINK-1", "LINK-2", "LINK-3",
+//                        "ISSUE-1", "ISSUE-2", "ISSUE-3",
+//                        "TMS-1", "TMS-2", "TMS-3"
+//                );
+//    }
```
Comment (sidelnikovmike) [inline allure-junit5/src/test/java/io/qameta/allure/junit5/AnnotationsTest.java]: Sorry :)

**Comment by sskorol** [comment]
Summon @baev

**Comment by sidelnikovmike** [comment]
@eroshenkoam or @baev could you look at PR?

**Comment by baev** [comment]
> Group - another class?

I mean running tests using tags

**Comment by baev** [comment]
@sidelnikovmike @sskorol thanks! 👍

**Pull request merged**